blocks/library/gallery/index.js

```diff
@@ -121,16 +124,24 @@ registerBlockType( 'core/gallery', {
 			controls,
 			focus && images.length > 1 && (
 				<InspectorControls key="inspector">
+					<p className="editor-block-inspector__description">Image galleries are a great way to share groups of pictures on your site.</p>
```

Comment:
I could create a PR for that if nobody is working on it yet. Sounds quite simple.